scss/components/_menu.scss

```diff
@@ -218,7 +273,7 @@ $menu-border: $light-gray !default;
     }
   }
   @else if $position == top {
-    > li > a {
+    li a {
```

Comment:
Why do we drop the `>` specifications? This feels likely to apply too broadly now...